File: thorium/src/test/integration/modif/testModif.py
import pandas as pd
table = pd.read_csv("data.csv")
table.loc[2] = ["Hugo",22,"Lyon","INFO"]
table.at[2, 3] = "GMA"
table.at[2, 'age'] = 40
table.to_csv("./src/test/integration/modif/generated.csv",index=False)

File: thorium/src/test/integration/modif/testModif.R
table <- read.csv("data.csv")
table[2,] <- c("Hugo",22,"Lyon","INFO")
table[2, 4] <- "GMA"
table[2, "age"] <- 40
write.csv(table, "./src/test/integration/modif/generated.csv", row.names=FALSE, quote=FALSE)
